app/src/main/java/net/squanchy/navigation/Navigator.java

```diff
@@ -26,51 +27,48 @@
 
 public class Navigator {
 
-    private final Context context;
+    private static final int NO_FLAGS = 0;
```

Comment:
shouldn't there be an empty line between `static final` fields and other fields?